cmd/snap/cmd_snap_op.go

```diff
@@ -369,9 +369,12 @@ func showDone(cli *client.Client, names []string, op string, opts *client.SnapOp
 				msg = fmt.Sprintf(i18n.G("%q left the cohort"), snap.Name)
 			}
 			fmt.Fprintln(Stdout, msg)
+			// go to next snap, skipping the temporary forward logic below for "switch" op.
+			continue
```

Comment:
I'm really sitting on the fence here - I don't really like the "continue" here as it feels a bit magic. I wonder if something like: http://paste.ubuntu.com/p/kGHWBFjJ97/ would be simpler? Maybe @chipaca has an opinion?